distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/InstallPluginCommandTests.java

```diff
@@ -822,26 +821,6 @@ public void testMissingDescriptor() throws Exception {
         assertInstallCleaned(env.v2());
     }
 
-    public void testMissingDirectory() throws Exception {
```

Comment:
I think it'd be nice if we had a nice error message for folks that install a plugin with the directory.

Reply:
I pushed 8b8bd7c